zcutil/fetch-params.sh

```diff
@@ -6,27 +6,88 @@ PARAMS_DIR="$HOME/.zcash-params"
 
 SPROUT_PKEY_NAME='sprout-proving.key'
 SPROUT_VKEY_NAME='sprout-verifying.key'
+
 SPROUT_PKEY_URL="https://zcash.dl.mercerweiss.com/$SPROUT_PKEY_NAME"
 SPROUT_VKEY_URL="https://zcash.dl.mercerweiss.com/$SPROUT_VKEY_NAME"
 
+#Using Zcash IPFS URL
+SPROUT_URL="https://myhush.org/#Tools"
+SPROUT_IPFS="/ipfs/QmZKKx7Xup7LiAtFRhYsE1M7waXcv9ir9eCECyXAFGxhEo"
+
 SHA256CMD="$(command -v sha256sum || echo shasum)"
 SHA256ARGS="$(command -v sha256sum >/dev/null || echo '-a 256')"
 
+WGETCMD="$(command -v wget || echo '')"
+IPFSCMD="$(command -v ipfs || echo '')"
+
+# fetch methods can be disabled with ZC_DISABLE_SOMETHING=1
```

Comment:
@radix42 we most likely should use our own environment variable here, right? Something like `HUSH_DISABLE_WGET` ?